rules/filename-case.js

```diff
@@ -3,6 +3,7 @@ var path = require('path');
 var camelCase = require('lodash.camelcase');
 var kebabCase = require('lodash.kebabcase');
 var snakeCase = require('lodash.snakecase');
+var pascalCase = require('pascal-case');
```

Comment:
Hmm, it was the one that I normally use but you're right. I'll update the deps and stick with Lodash only.